docs/requirements.md

```diff
@@ -22,8 +22,8 @@ The cluster running Codacy must satisfy the following requirements:
 
 -   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
-    -   [Kubernetes](https://kubernetes.io/) **version 1.14.\*** to **1.20.\*** (1.20 recommended)
-    -   [MicroK8s](https://microk8s.io/) **version 1.14.\*** to **1.19.\*** (1.19 recommended)
+    -   [Kubernetes](https://kubernetes.io/) **version 1.15.\*** to **1.20.\*** (1.20 recommended)
```

Comment:
Shouldn't it be "to 1.21"?

Reply:
We're just testing in microk8s for now, which is fairly limited. I see it as a first test towards fully supporting 1.21, which will require further testing, and eventually using it ourselves.